README.md

```diff
@@ -16,12 +16,13 @@ Effectively any significant change to the state of the network, requiring author
 
 Top level direct names should be the [EIP 3770](https://eips.ethereum.org/EIPS/eip-3770) short name for the network (see [shortNameMapping.json](https://chainid.network/shortNameMapping.json))
 
-Each task will have a directory structure similar to the following:
+Each task should contain the following:
 
-- `task-name.json`: A json file which defines the task to be executed. This file may either be generated automatically or manually created.
+- `README.md`: A brief markdown file describing the task to be executed.
+- `Validation.md`: A markdown file describing and justifying the expected state changes for manual validation.
+- `input.json`: A json file which defines the task to be executed. This file may either be generated automatically or manually created.
+- `diff.json` (optional): A json file enabling the automated verification of the expected state diff.
```

Comment:
Can you expand on this? I'm not familiar with and don't see one in the repo

Reply:
This is the planned format that will be read by the state diff checker toling.

Reply:
Removing just to obviate further discussion for now.

```suggestion
```